src/chrome/content/rules/Brightcove.xml

```diff
@@ -103,6 +103,7 @@
 <ruleset name="Brightcove (partial)">
 
 	<target host="*.brightcove.com" />
+	<target host="go.brightcove.com" />
```

Comment:
That's already covered by the wildcard. But this rule can be reqritten without using a wildcard anyway (if you want to create a new PR for this ;)).

Reply:
I intended that as a first step to remove the wildcard but perhaps that's not so useful. Should I remove this target?

Reply:
Yes, duplication is worse.